documentation/src/main/asciidoc/topics/proc_configuring_hotrod_percache.adoc

```diff
@@ -0,0 +1,47 @@
+[id='hotrod_percache']
+= Per-cache configuration
+
+Certain aspects of remote caches can be configured for each individual cache:
+
+* force return values
+* on-access creation configuration/template
+* near-caching
+* transaction mode
+
+.Procedure
+
+* Enable _force return values_ just for cache `a-cache`
+
+[source,java]
+----
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+...
+
+ConfigurationBuilder builder = new ConfigurationBuilder();
+builder
+  .remoteCache("a-cache")
+    .forceReturnValues(true);
+----
+
+You can also use wildcard globbing in the remote cache name to match multiple caches:
+
+.Procedure
+
+* Enable _force return values_ for all caches starting with the string `somecaches`.
+
+[source,java]
+----
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+...
+
+ConfigurationBuilder builder = new ConfigurationBuilder();
+builder
+  .remoteCache("somecaches*")
+    .forceReturnValues(true);
+----
+
+[NOTE]
+====
+When using declarative configuration and your cache names contain the `.` character, you must surround the cache name
+with enclosing square brackets, e.g. `infinispan.client.hotrod.cache.[example.MyCache].template=...`
+====
```

Comment:
Apologies in advance because this is probably obvious but I question the "When using declarative configuration" part here. Does this mean that if caches defined in infinispan.xml contain the `.` character in the name? Or is this more about if you use hotrod-client.properties and caches contain the `.` character? 

Also, no Latin abbreviations please. :smile: s/e.g./for example:/

Another nit that I can't hold back, surround and enclose pretty much mean the same thing. It might be more efficient to say "you must enclose the cache name in square brackets"

Reply:
This is about the `hotrod-client.properties` file, since the `.` character is already a delimiter in the property name

Reply:
I've addressed the comments